### DISCUSSION TITLE
Access old account

### DISCUSSION THREAD

**Discussion by captray** [community/community#121835] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

A few years ago (probably morel like 2017) I purchased a new phone and lost a lot of recovery codes to multiple products (GitHub, LinkedIn, et al). While I was able to recover most of them, I was not able to get back into my GitHub account. The deal that the support agent made with me back then was that if no one logged into my old account (captray), which he marked "captray-zz", then I would be able to merge the accounts or something along those lines. I would like to get access to this account again as there are a couple of repos there I would like to retain control over, update, etc. Please, let me know what the next steps are. Thanks!

**Comment by ebndev** [comment, marked as answer]
Hi @captray,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
